py_pdf_parser/loaders.py

```diff
@@ -69,7 +70,9 @@ def load(
     la_params = {**DEFAULT_LA_PARAMS, **la_params}
 
     pages: Dict[int, Page] = {}
-    for page in extract_pages(pdf_file, laparams=LAParams(**la_params)):
+    for page in extract_pages(
+        pdf_file, laparams=LAParams(**la_params), password=password
```

Comment:
Please could you add the new `password` parameter into the docstring of this function?

Reply:
@jstockwin Done!